definitions/tools/sequence_align_and_tag.cwl

```diff
@@ -45,14 +47,31 @@ requirements:
                     n)
                         NTHREADS="$OPTARG"
                         ;;
+                    t)
+                        TRIMMING_ADAPTERS="$OPTARG"
+                        ;;
+                    o)
+                        TRIMMING_ADAPTER_MIN_OVERLAP="$OPTARG"
+                        ;;
                 esac
             done
 
             if [[ "$MODE" == 'fastq' ]]; then
-                /usr/local/bin/bwa mem -K 100000000 -t "$NTHREADS" -Y -R "$READGROUP" "$REFERENCE" "$FASTQ1" "$FASTQ2" | /usr/local/bin/samblaster -a --addMateTags | /opt/samtools/bin/samtools view -b -S /dev/stdin
+                if [[ -z \${TRIMMING_ADAPTERS-} || -z \${TRIMMING_ADAPTER_MIN_OVERLAP-} ]]; then
+                    /usr/local/bin/bwa mem -K 100000000 -t "$NTHREADS" -Y -R "$READGROUP" "$REFERENCE" "$FASTQ1" "$FASTQ2" | /usr/local/bin/samblaster -a --addMateTags | /opt/samtools/bin/samtools view -b -S /dev/stdin
+                else
+                    /opt/flexbar/flexbar --adapters "$TRIMMING_ADAPTERS" --reads "FASTQ1" --reads2 "$FASTQ2" --adapter-trim-end LTAIL --adapter-min-overlap "$TRIMMING_ADAPTER_MIN_OVERLAP" --adapter-error-rate 0.1 --max-uncalled 300 --stdout-reads \
```

Comment:
Just wanna make sure two hardcoded parameters "--adapter-error-rate 0.1" and "--max-uncalled 300" are the optimized ones for most cases.